packages/components/src/DateTimePickers/InputDateTimeRangePicker/InputDateTimeRangePicker.snapshot.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import InputDateTimeRangePicker from './InputDateTimeRangePicker.component';
+
+describe('InputDateTimeRangePicker', () => {
```

Comment:
Can you move this test in the other test file please ?

Reply:
yes sure.